python/tests/test_fs.py

```diff
@@ -184,13 +185,6 @@ def test_roundtrip_azure_env(azurite_env_vars, sample_data: pa.Table):
 def test_roundtrip_azure_direct(azurite_creds, sample_data: pa.Table):
     table_path = "az://deltars/roundtrip2"
 
-    # Fails without any creds
-    with pytest.raises(PyDeltaTableError):
-        anon_storage_options = {
-            key: value for key, value in azurite_creds.items() if "ACCOUNT" not in key
-        }
-        write_deltalake(table_path, sample_data, storage_options=anon_storage_options)
```

Comment:
Not entirely sure what we did before, but now `AZURE_STORAGE_USE_EMULATOR` will pick up the default account / key if none are specified - so its hard to create a failing state with the emulator now.